src/main/java/fmartin1/model/pokemon/type/TypePokemon.java

```diff
@@ -0,0 +1,18 @@
+package fmartin1.model.pokemon.type;
+
+
+import com.fasterxml.jackson.annotation.JsonIgnoreProperties;
+import fmartin1.model.common.NamedAPIResource;
+
+@JsonIgnoreProperties(ignoreUnknown = true)
+public class TypePokemon {
+    private final NamedAPIResource _pokemon;
```

Comment:
Maybe a better name would be: `_apiPokemon` to differentiate from your `Pokemon` model